src/Carousel.vue

```diff
@@ -546,6 +611,20 @@ export default {
       }
     },
     handleNavigation(direction) {
+      if (this.infinite) {
+        // A timeout is needed for the button press to control the animations
+        if (!this.navigationButtonPressed) {
+          this.navigationButtonPressed = true;
+          setTimeout(
+            function() {
+              this.navigationButtonPressed = false;
+            }.bind(this),
+            750
```

Comment:
General question about the timeout lengths chosen, is there a specific reason it was set to the value? I see that there was a different timeout chose for most of the setTimeouts that were used.

Reply:
These lengths are set more to a feeling than anything else, giving the DOM enough time to move things around on somewhat slow machines as well. I also believe that in general a async way would have been better, but it would have taken longer to make. 

I also believe that the way this carousel is set up isn't really good for making a carousel infinite.

Reply:
Yeah, I get that feeling as well. I think to properly implement this it would take a bit of a rewrite and I get that you don't have that time. I'll have to discuss things with @quinnlangille, maybe we can release this feature as an experimental one and then go from there.

Reply:
Hey, chatted with @ashleysimpson about this. I'm fine merging into an experimental branch but as is it's not ready for production. @MartinGerritsen do you think you'll be able to implement the requested changes? Totally fine if not - just let us know. 

Since this feature is pretty sought after, we'll likely include some version of it in the upcoming v1 release. Would be great to have your contribution included, but if time commitment is an issue then we'll likely delivery the feature internally. Lemme know what works for you! :octocat: